client/components/like-button/button.jsx

```diff
@@ -83,8 +83,14 @@ var LikeButton = React.createClass( {
 
 		containerClasses = classnames( containerClasses );
 
+		if ( likeCount === 0 && ! this.props.showCount ) {
+			likeCount = '';
+		} else {
+			likeCount = likeCount + ' ';
```

Comment:
I don't want there to always be a trailing space, only when likeCount is not 0 and we're showing the count